internal.nix

```diff
@@ -295,7 +295,14 @@ rec {
   # Type: Set -> Set -> Set | null
   sourceHashFunc = githubSourceHashMap: spec:
     if spec.type == "github" then
-      lib.attrByPath [ spec.value.org spec.value.repo spec.value.rev ] null githubSourceHashMap
+      lib.attrByPath
+      [ spec.value.org spec.value.repo spec.value.rev ]
+      (
+        lib.traceSeq
```

Comment:
Can you please ensure that these messages have a `npmlock2nix:` prefix? Just like we do with our custom throw wrapper: https://github.com/nix-community/npmlock2nix/blob/1e373fb0061ef64f9b99288601d5c9a98a5522a0/internal.nix#L20-L22

This helps debugging larger builds where it might not be obvious where the warning might have been generated at.